dnf/base.py

```diff
@@ -64,6 +64,7 @@
 import re
 import rpm
 import time
+import shutil
```

Comment:
There is incorrect bug number, It should be: 1279001 , but your number is nicer :-) .